build.sbt

```diff
@@ -16,5 +16,6 @@ parallelExecution in Test := false
 
 libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-  "com.novocode" % "junit-interface" % "0.11" % "test"
+  "com.novocode" % "junit-interface" % "0.11" % "test",
+  "com.storm-enroute" %% "scalameter" % "0.8.2"
```

Comment:
I'm not aware of any issues. I don't know why we use sbt-jmh instead. Maybe for historical reasons.